ToolBox/TradingEconomicsDataDownloader/TradingEconomicsCalendarDownloader.cs

```diff
@@ -180,52 +115,61 @@ private bool ProcessData(List<TradingEconomicsCalendar> data)
             var status = true;
 
             Parallel.ForEach(
-                data.GroupBy(x => GetTicker(x.Ticker, x.Category, x.Country)),
+                data.GroupBy(x => x.Country.ToLowerInvariant()),
```

Comment:
Is `x.Country` not normalized that we requrie `.ToLowerInvariant()`? Think it should be normalized/always the same

Reply:
There are instances where the country name could differ in capitalization

Reply:
It would be nice if it was normalized, all upper case maybe? Same for the Symbols `public const string InterestRate = "United-States//FDTR"` -> "UNITED-STATES//FDTR" - and same for the data path 🤔

Reply:
Nevermind the data path 👍  https://github.com/QuantConnect/Lean/pull/4054#issuecomment-582371867